client/src/pages/european-projects/components/pages/analysis/positioning/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Container } from "@dataesr/dsfr-plus";
+import { useSearchParams } from "react-router-dom";
+import { useEffect } from "react";
+import Top10Beneficiaries from "./charts/top-10-beneficiaries";
+
+export default function Positioning() {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  useEffect(() => {
+    if (searchParams.get('country_code') === null) {
+      setSearchParams({ country_code: 'FRA' });
```

Comment:
searchParams.append('country_code': 'FRA');
setSearchParams(searchParams);

I do not know if you need to do it through a copy of searchParams ?